transcom/mymove
# B-21597: Add Temporary Duty (TDY) Order Type INT

## [B-21597](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21597)

## Summary

The orders type of TDY vs PCS will influence differences in UB (unaccompanied baggage) shipment weight allowances. So, we needed to add the Temporary Duty (TDY) type to the orders type options.

Work in this PR:
- Made the orders_type an enum
- Added TEMPORARY_DUTY as an orders_type enum value
- Updated tests to use constants instead of hard-coded values for orders type, where it made sense to do so

To Test:
1. Create a new customer and get to the point of adding orders
2. Confirm that Temporary Duty (TDY) is an option to select for orders
3. Select TDY as your orders type
4. Continue through adding a few shipments and submit the move
5. Confirm the orders were created with orders_type of TEMPORARY_DUTY
6. Run through the SC and TOO process and keep an eye out for any oddities

<img width="713" alt="image" src="https://github.com/user-attachments/assets/186fe84f-20ba-449c-a470-9071b01e7f04">

<img width="403" alt="image" src="https://github.com/user-attachments/assets/010b7b35-04f4-438b-85cd-6137322737c3">

<img width="511" alt="image" src="https://github.com/user-attachments/assets/0351c646-8965-4fa1-80ad-d7a384a409b0">

<img width="371" alt="image" src="https://github.com/user-attachments/assets/e6fb2ce5-cc34-49e1-8f52-703ed3f8e7a4">

<img width="416" alt="image" src="https://github.com/user-attachments/assets/264e7f6e-60ee-40f7-a3b9-668a3bba6960">

508: 
<img width="1231" alt="image" src="https://github.com/user-attachments/assets/f32e926e-f7d1-4601-8456-9295fa6edad6">
